package.json

```diff
@@ -14,6 +14,7 @@
     "packages/dual-publish",
     "packages/size-limit",
     "packages/webpack",
+    "packages/webpack-extender",
```

Comment:
What do you think about `webpack-config` name to specify that we are extending Webpack config?